README.rst

```diff
@@ -6,11 +6,9 @@ BenchOpt is a package to simplify and make more transparent and reproducible
 the comparisons of optimization algorithms. This benchmark is dedicated to
 benchmarking algorithms that solve the full lasso path, that is, solving
 
-.. math::
+$$ \\min_w ~ \\frac{1}{2} \\|y - Xw\\|^2_2 ~+~ \\lambda \\|w\\|_1 $$
```

Comment:
```suggestion
$$\\min_w \\left(\\frac{1}{2} \\|y - Xw\\|^2_2 + \\lambda \\|w\\|_1\\right)$$
```
I don't like messing with spacing manually, so I prefer to avoid the `~`. I also added parenthesis for additional clarity.